client/routes/pageNotFound.html

```diff
@@ -1,5 +1,9 @@
 <template name="pageNotFound">
-  <h1>{{_ "Error_404"}}</h1>
-  <h3>{{_ "Oops_page_not_found"}}</h3>
-  <p>{{_ "Sorry_page_you_requested_does_not_exists_or_was_deleted"}}</p>
+	<section class = "page-not-found-container content-background-color">
+		<span class = "error-404">{{errorcode}}</span>
```

Comment:
I don't think you need a helper here. Since the template's name is `pageNotFound`, it will be only applied for HTTP 404 cases.

Reply:
Yes, you are right...I did not overlook that